### DISCUSSION TITLE
All my projects and teams are gone...after trying to update my account email.  Please help!

### DISCUSSION THREAD

**Discussion by coffee-converter** [vercel/community#1363] [Help] [answered]
### Summary

I had always logged in to vercel via my github account.  Today I noticed that an old employer's email was the one on my vercel account, so I went to change it to my gmail address, which is also the one associated with my github account.  Vercel's UI wouldn't let me change the email, it said "this email is unavailable".  This led me to believe it might be noticing that my github associated to my gmail was already connected.  So, I logged out, and logged in via my gmail.  I noticed that I no longer saw any of my teams or projects.  So, I logged out, and logged back in via github as usual, but now I again noticed no teams or projects!  The old company email no longer exists, as I no longer work for that company.  How can I migrate my old teams/projects to my account again?  Please help, I am unable to do any work now!

**Comment by HugaidaS** [comment]
I have the same issue and unable to change email to my own now, seems like a stalemate, I lost access to all my deployed project, and I also can't login with email with my phone number cz it was attached to my github login....

**Comment by HugaidaS** [comment]
I also found this warning on the instructions page 

WARNING: If your new email address differs from your Git account's primary email, you will not be able to log in to your Vercel personal account.

don't you think you should notify users about it when changing email in settings, and not somewhere on another page? I am extremely disappointed.

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account info, we handle this type of issue via a support ticket. Please open a new case by sending an email to [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the accounts. That team has access to look up account info associated with your old email address to help you get this sorted. 🙇

**Comment by andre-rd-rodrigues** [comment]
I'm having the exact same issue. I don't have access to my original account projects, cause I've registered a new account and connected my Gibhub account. So frustrating. I've sent email to [registration@vercel.com](mailto:registration@vercel.com).

**Reply by amyegan** [reply]
Hey @andre-rd-rodrigues. You should be able to access your other account by logging in with email instead of GitHub
